yaml2rst/resources/excluded_key.csv

```diff
@@ -0,0 +1 @@
+uniqueItems,additionalProperties,
```

Comment:
This could be specified on the CLI and have the above as a default, so that client code could decide to blacklist even more.